app/src/main/java/lapuk_app/views/main/MainActivity.kt

```diff
@@ -82,7 +83,9 @@ fun MainScreen() {
                             TODO()
                         }
                         composable("info") {
-                            TODO()
+                            composable("privacy-policy") {
```

Comment:
i think another item ni sa navcontroller di siya inside sa infocenter na composable, pero to define na under siya sa infocenter dapat composable("info/privacy-policy")